app/etc/di.xml

```diff
@@ -1756,4 +1756,61 @@
             </argument>
         </arguments>
     </type>
+    <preference for="\Magento\Framework\GraphQl\Query\ErrorHandlerInterface" type="\Magento\Framework\GraphQl\Query\ErrorHandler"/>
```

Comment:
Looks like we need to move to `graphql` section

Reply:
It is possible to specify the area only in module scope, am I right?
Unfortunately, this code is placed in `lib/internal`, so I can not see the solution.

Could you assist, please?

Reply:
@novikor - Check it out `lib/internal/Magento/Framework/App/Utility/Files.php:1331` - putting di.xml into app/etc/graphql should do the trick.